src/java/org/apache/cassandra/locator/TokenMetadata.java

```diff
@@ -281,14 +281,6 @@ public void updateAddressForNormalTokens(Collection<Token> tokens, InetAddressAn
         lock.writeLock().lock();
         try
         {
-            Collection<Token> oldNodeTokens = tokenToEndpointMap.inverse().get(existing);
```

Comment:
Any specific reasons you removed this? It did make sense to me, as the objective is to update the address but keep the tokens, otherwise it means the node must re-join.

Reply:
I assumed that throwing exception in `TokenTracker` makes the system unstable, but "token mismatch" should not happen in practice. I will add the exception back.